### ISSUE TITLE
consider keybindings that don't conflict with JAWS 

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#215578]
@jooyoungseo mentioned that the current keybinding for this controls speech rate for jaws users on windows
`ctrl+pagedown / ctrl+pageup` should be used for code chat next code block and go to next /previous cell

**Comment by amunger** [comment]
related https://github.com/microsoft/vscode/issues/176290.  ctrl+pagedown/pageup are currently bound to next/previous editor

**Comment by meganrogge** [comment]
@amunger shouldn't those have context keys though that will mean this can still work for notebook cells?

**Comment by amunger** [comment]
next/previous editor don't have context keys for those keybindings, so it will be a behavior change.
We could add the bindings just for SR mode if you don't think those users would be annoyed by it.

**Comment by meganrogge** [comment]
@jooyoungeso, what do you think? I also don't want to surprise users with this behavior change.

**Comment by jooyoungseo** [comment]
I don’t think this is a surprise change. If we weigh the tradeoff, going with the new keybindings avoiding SR keys might be more beneficial in the long run.
